content/blog/2016/2016-05-10-jenkins-20-vjam.adoc

```diff
@@ -0,0 +1,355 @@
+---
+layout: post
+author: rtyler
+title: "Jenkins 2.0 Virtual JAM Wrap-up"
```

Comment:
As you remember "Virtual JAM" has been rejected by the hosting. I would stick to the "Online Meetup" at least within the public materials